src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -194,7 +194,7 @@ def elide_two(text1, text2, overall_length = 40, mode = :middle)
   end
 
   def force_utf8_and_transform_nonprintables(text)
-    text.force_encoding('UTF-8')
+    text.to_s.force_encoding('UTF-8')
```

Comment:
I think the next line would still fail when text is nil: `unless text.valid_encoding?`

Reply:
I also included a fix in my PR
https://github.com/openSUSE/open-build-service/pull/5943/commits/61fa16ff2b191a8eaf6c9f6ed991a5583db51265

Let me know if this gets merged so I can drop and rebase or we close this one and take my PR instead...